Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -150,6 +150,32 @@ function rotateData(obj,axis,value)
             obj.rotateTable(obj.forces, axis, value);
             disp('Marker and Force tables have been rotated')
         end
+        function convertMillimeters2Meters(obj)
```

Comment:
Can we rename obj to osimC3Dobject for clarity?

Reply:
As far as I am aware, it can only be obj (obj is itself).

Reply:
So on line 40 we can't name the object as c3dObject for example?? I guess then users are accustomed to confusion! My bad

Reply:
I checked out some documentation from a few different sources, and you are correct, I can change this into a custom name. However, all examples I had ever seen used _obj_ , with some of the examples I found today using _self_. So I am hesitant to change it since someone new to OOP in Matlab may nor realize the mapping.

Reply:
Also can we verify the units instead of assuming them to avoid potentially corrupting data or are the units not built into the format?

Reply:
I would prefer 'self' which means something special over a generic 'obj' that can easily get lost, but you're call. Thanks for investigating.

Reply:
I agree, _self_ feels like it would be a better differentiator than _obj_. I had always seen _obj_ in examples so went with that. I will update to _self_ here since it doesn't break anything (I tested it locally already).

Reply:
> Also can we verify the units instead of assuming them to avoid potentially corrupting data or are the units not built into the format?

Agreed, but there isn't information on units in the Forces table metadata.  There is probably (almost certainly) a way of getting it from _C3DFileAdapter()_ and including it in the metadata.

The reasons for exposing _convertMillimeters2Meters()_ was actually because of this. I had been converting everyones data by default at write time, which is probably fine in most cases (Vicon stores in mm) but it could cause havoc on if not.